configure.ac

```diff
@@ -976,6 +976,18 @@ AC_SUBST(JEMALLOC_LDFLAGS)
 AC_SUBST(JEMALLOC_AUTOCONF_FLAGS)
 fi
 
+
+AC_ARG_WITH(crash_privacy,       [  --with-crash_privacy=yes,no         If you want your crashes to not include names of symbols present in the binary. ], [], [with_crash_privacy=no])
```

Comment:
It might be better to avoid underscores in these option names and use - instead. Its easier to type, looks better, and consistent with autoconf conventions.

Reply:
Autoconf makes the two completely indistinguishable. You can use a dash or an underscore when passing the option and it'll hit this same option. I use underscores for consistency because we use underscores elsewhere.

Reply:
Autoconf also allows single dash which I think is preferable for brevity.
The odds of single character options colliding is slow. They often don't even exist. Or of them being file names.

Reply:
@alexanderkyte we mostly use `-` over `_` in configure options, please respect this convention.